numpy/core/src/umath/reduction.c

```diff
@@ -436,6 +437,7 @@ PyArray_InitializeReduceResult(
  * loop        : The loop which does the reduction.
  * data        : Data which is passed to assign_identity and the inner loop.
  * buffersize  : Buffer size for the iterator. For the default, pass in 0.
+ * errormask   : forwarded from _get_bufsize_errmask
```

Comment:
Since it comes after `funcname` in the signature, move it below it here as well.